README.md

```diff
@@ -15,17 +15,30 @@ npm install @edgepi-cloud/node-red-edgepi-digital-out
 - **RPC Server**<br> 
 The connection to your EdgePi's RPC Server.
 - **Channel**<br>
-The channel you want to configure.
+The channel being configured.
 - **State**<br>
-The state you want to set the selected channel to.
+The state to set the selected channel to.
 
 ### Inputs
-Any message can be used to trigger this node.
+- **payload** *string*<br>
+The D-OUT state.
+- **channel** *number*<br>
+
+Example input:
+```
+msg {
+    "payload": "LOW",
+    "channel": 2
+}
```

Comment:
provide possible options for state value.

Reply:
Fixed.